tasks/sep/028-holocene-system-config-upgrade-and-init-base/input.json

```diff
@@ -0,0 +1,135 @@
+      "value": "0",
+      "contractMethod": {
+        "inputs": [
+          {
+            "name": "_proxy",
+            "type": "address",
+            "internalType": "address payable"
+          },
+          {
+            "name": "_implementation",
+            "type": "address",
+            "internalType": "address"
+          },
+          {
+            "internalType": "bytes",
+            "name": "_data",
+            "type": "bytes"
+          }
+        ],
+        "name": "upgradeAndCall",
+        "payable": false,
+        "outputs": []
+      },
+      "contractInputsValues": {
+        "_data": "0xdb9040fa0000000000000000000000000fe884546476ddd290ec46318785046ef68a0ba9000000000000000000000000000000000000000000000000000000000000044d00000000000000000000000000000000000000000000000000000000000a118b000000000000000000000000fc56e7272eebbba5bc6c544e159483c4a38f8ba30000000000000000000000000000000000000000000000000000000003938700000000000000000000000000b830b99c95ea32300039624cb567d324d4b1d83c0000000000000000000000000000000000000000000000000000000001312d00000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000003b9aca0000000000000000000000000000000000000000000000000000000000000f424000000000000000000000000000000000ffffffffffffffffffffffffffffffff000000000000000000000000ff00000000000000000000000000000000084532000000000000000000000000c34855f4de64f1840e5686e64278da901e261f2000000000000000000000000021efd066e581fa55ef105170cc04d74386a09190000000000000000000000000fd0bf71f60660e2f608ed56e1659c450eb113120000000000000000000000000d6e6dbf4f7ea0ac412fd8b65ed297e64bb7a06e100000000000000000000000049f53e41452c74589e85ca1677426ba426459e85000000000000000000000000b1efb9650ad6d0cc1ed3ac4a0b7f1d5732696d37000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee",
+        "_proxy": "0xf272670eb55e895584501d564AfEB048bEd26194",
+        "_implementation": "0x33b83E4C305c908B2Fc181dDa36e230213058d7d"
+      }
+    },
+    {
+      "metadata": {
+        "name": "SystemConfigProxy.setEIP1559Params(1,4)",
+        "description": "Set the EIP1559 `_denominator` to `1` and `_elasticity` to `4`"
+      },
+      "to": "0xf272670eb55e895584501d564AfEB048bEd26194",
```

Comment:
I think this is the address for Base Sepolia. You want the address from here https://github.com/ethereum-optimism/superchain-registry/blob/cd93875324ef417e4932b72a0d27daa16a552202/superchain/configs/mainnet/base.toml#L59